Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM gcc:latest as build
+
+WORKDIR /app
+
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y cmake
+
+COPY . /app/
+
+RUN sed -i 's/MARIADB_CONNECTOR_COMPILE_JOBS__=1/MARIADB_CONNECTOR_COMPILE_JOBS__=2/' /app/CMakeVariables.txt 
+
+RUN mkdir build
+RUN cd build && \
+	cmake .. && \
+	make -j2
+
+FROM gcc:latest as runtime
+
+WORKDIR /app
+
+COPY --from=build /app/build/*Server /app/
+COPY --from=build /app/build/*.ini /app/default-configs/
+COPY --from=build /app/build/*.so /usr/lib/
```

Comment:
While this is technically correct, without another fix, this will be missing the `libmariadb.so`.